app/components/chart/footer.js

```diff
@@ -74,28 +77,41 @@ var TidelineFooter = React.createClass({
             onChange={this.props.onClickLines} /> Lines
         </label>
       </div>
-      );
-    
+    );
 
-    
-    var rightSide = this.props.chartType === 'weekly' ? showValues :
-      this.props.chartType === 'modal' ? modalOpts : null;
-    
+    var rightSide = null;
+    var bgDataToggle = null;
+
+    if (this.props.chartType === 'weekly') {
+      rightSide = showValues;
+    } else if (this.props.chartType === 'modal') {
+      if (this.props.showingSmbg) {
+        rightSide = modalOpts;
+      }
+      bgDataToggle = (
+        <TwoOptionToggle
+          left={{ label: 'BGM', state: this.props.showingSmbg }}
+          right={{ label: 'CGM', state: this.props.showingCbg }}
+          toggleFn={this.props.onClickBgDataToggle}
+        />
+      );
+    }
 
-    
     return (
       <div className="container-box-outer patient-data-footer-outer">
         <div className="container-box-inner patient-data-footer-inner">
           <div className="patient-data-footer-left">
             <button className="btn btn-chart btn-refresh"
               onClick={this.props.onClickRefresh}>
               Refresh</button>
+            <span className="toggle-container">
```

Comment:
Would it make sense to move this wrapping `<span>` into the `bgDataToggle` instead of having it hang out in the DOM when it has no contents?